### PR TITLE
#warning -> #warn, episode 2, season finale

### PR DESCRIPTION
**Pull request by DamianX**

### CODE REVIEW

File: code/datums/feedback.dm [hunk 1]
```diff
@@ -67,7 +67,7 @@ var/datum/blackbox/blackbox = new
 	if(!feedback)
 		return
 
-	//#warning Blackbox recording disabled.  Please remove warning once this has been determined to be the problem.
+	//#warn Blackbox recording disabled.  Please remove warning once this has been determined to be the problem.
```
Comment (MadmanMartian) [inline code/datums/feedback.dm:70]: lmao

File: code/defines/procs/biohazard_alert.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@ var/global/list/outbreak_level_words=list(
 	command_alert(/datum/command_alert/biohazard_alert)
 
 /*
-#warning TELL N3X15 TO COMMENT THIS SHIT OUT
+#warn TELL N3X15 TO COMMENT THIS SHIT OUT
```
Comment (MadmanMartian) [inline code/defines/procs/biohazard_alert.dm:14]: lmao

File: code/modules/mob/living/simple_animal/borer.dm [hunk 3]
```diff
@@ -1040,7 +1040,7 @@ var/global/borer_unlock_types_leg = typesof(/datum/unlockable/borer/leg) - /datu
 			//testing("Client of [G] inexistent")
 			continue
 
-		//#warning Uncomment me.
+		//#warn Uncomment me.
```
Comment (MadmanMartian) [inline code/modules/mob/living/simple_animal/borer.dm:1043]: lmao

Comment (Exxion) [reply code/modules/mob/living/simple_animal/borer.dm:1043]: I honestly cannot comprehend what the fuck this line hoped to achieve

**Pull request merged**